### PR TITLE
BaSP-MR-53: Create admin list/delete table 

### PR DESCRIPTION
**Pull request by AleMac17**
> Show table with admin list with its respective columns
> Add delete row functionality
> Add an actions icon (delete and edit) for each row
> Warning message for delete
> Add "add new Admin" button and redirect to add form

### CODE REVIEW

**Review by lorenzomauro59 — APPROVED** [review]
It's approved!

**Comment by fernandofernandezuy** [comment]
I see you have some files with the termination .jsx and others with .js, we must have consistency across the repo

**Review by fernandofernandezuy — APPROVED** [review]
Looks good to me!

**Comment by Gonzarey96** [comment]
[[Members] List/Delete table](https://trello.com/c/FyRrK72k/61-members-list-delete-table)

**Comment by AleMac17** [comment]
[[Admin] List/Delete table](https://trello.com/c/EZavij9r/53-admin-list-delete-table)

File: src/Components/Admins/Modals/modal.module.css
Comment (nailaghigo) [inline src/Components/Admins/Modals/modal.module.css:1]: We should fix props indentations inside the entire file

File: src/Components/Admins/Table/table.module.css
Comment (nailaghigo) [inline src/Components/Admins/Table/table.module.css:1]: Fix indentations here too

File: src/Components/Admins/admins.module.css
Comment (nailaghigo) [inline src/Components/Admins/admins.module.css:1]: Fix indentations here too

File: src/Components/Admins/Modals/index.jsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import styles from './modal.module.css';
+
+function Modal(props) {
+  if (!props.show) {
+    return null;
+  }
+
+  const onCloseModal = () => {
```
Comment (nailaghigo) [inline src/Components/Admins/Modals/index.jsx]: We should replace this name by something like `onConfirmDelete`

File: src/Components/Admins/index.jsx [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+  const deleteItem = (id) => {
+    setShowModal(true);
+    setIdDelete(id);
+  };
+
+  const closeModal = () => {
+    setShowModal(false);
+  };
+
+  const onCloseModal = () => {
```
Comment (nailaghigo) [inline src/Components/Admins/index.jsx]: We should rename this as `handleConfirmDelete`

File: src/Components/Admins/index.jsx [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+  useEffect(() => {
+    getAdmins();
+  }, []);
+
+  const deleteItem = (id) => {
+    setShowModal(true);
+    setIdDelete(id);
+  };
+
+  const closeModal = () => {
```
Comment (nailaghigo) [inline src/Components/Admins/index.jsx]: We should rename this as `handleCancelDelete`

File: src/Components/Admins/index.jsx [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+    setShowModal(false);
+  };
+
+  const onCloseModal = () => {
+    deleteAdmins(idDelete);
+    setAdmins([...admins.filter((admin) => admin._id !== idDelete)]);
+    setIdDelete();
+  };
+
+  if (admins != '') {
```
Comment (nailaghigo) [inline src/Components/Admins/index.jsx]:
```suggestion
  if (admins !== '') {
```

File: src/Components/Admins/index.jsx [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+  const deleteAdmins = async (id) => {
+    try {
+      const response = await fetch(`${process.env.REACT_APP_API_URL}/admins/${id}`, {
+        method: 'DELETE'
+      });
+      if (!response.ok) {
+        throw new Error('Error deleting admin.');
+      }
+      const data = await response.json();
+      console.log(data);
```
Comment (nailaghigo) [inline src/Components/Admins/index.jsx]: We should delete this console.log()

**Review by nailaghigo — APPROVED** [review]
Need some modifications 😸

**Review by LucianoAlarcon — APPROVED** [review]

**Comment by nazarenapantanetti** [comment]
Tested and approved!

**Pull request merged**